### DISCUSSION TITLE
Clip one .nc to extent of another

### DISCUSSION THREAD

**Discussion by albertlarson** [pydata/xarray#4910] [General]
I have two .nc files (a and b) with some overlapping featurespace. Is it possible to create a new version of (a) limited to the featurespace (lat,long) overlapping with (b)? Thank you!

**Comment by mathause** [comment]
Have a look at `align` and its join option: http://xarray.pydata.org/en/stable/generated/xarray.align.html#xarray.align

**Reply by albertlarson** [reply]
I just uploaded a two sample pieces of data and a notebook, linked below. I know that the datasets are arrays of two different sizes. The AMSR-E .nc file is a much larger swath than MODIS. When I attempt to combine them, it comes up with an error:

_ValueError: arguments without labels along dimension 'ni' cannot be aligned because they have different dimension sizes: {1354, 243}_

I tried using different join options, but it still kicks out an error regardless. Any thoughts? Thanks for the help!

https://github.com/albertlarson/xarray-21421

**Reply by mathause** [reply]
Ah yes this does not work for `Dimensions without coordinates`. Do you expect the lat & lon coordinates to be exactly the same (for a subset of them) or do you just want a similar extent, e.g. for two plots? 

This is much more difficult because you have 2D lat and lon coordinates & I don't normally work with such coord to know how I'd do this.
